src/ngMessages/messages.js

```diff
@@ -31,7 +31,9 @@ var jqLite = angular.element;
  *
  * ```html
  * <form name="myForm">
- *   <input type="text" ng-model="field" name="myField" required minlength="5" />
+ *   <label>Enter text:
+ *     <input type="text" ng-model="field" name="myField" required minlength="5" />
+ *   </label>
  *   <div ng-messages="myForm.myField.$error">
```

Comment:
`ngAria` actually adds them to `ngMessages`, but it can't hurt to include the role here.